content/geodesic/module/_index.md

```diff
@@ -5,3 +5,49 @@ description: "Geodesic modules are Git repositories that extend the Geodesic bas
 {{% dialog type="info" icon="fa fa-book" title="Read More" %}}
 ⇠ Select one of these pages to learn more about the topic.
 {{% /dialog %}}
+
+We believe that infrastructure as code should be treated just like all other code: tested, bundled into artifacts and automatically deployed. This is what `geodesic` solves. A geodesic module is a docker container/image contains all the code necessary deploy infrastructure.
+
+For example, when we need to provision resources using `terraform`, we'll build a docker container that inherits from the `geodesic` base image and `COPY` all the `*.tf` code into the image on `docker build`. Now we can run that container to provision infrastructure anywhere we have `docker` installed (for example from CI/CD systems), without needing to configure, version and maintain a bunch of native tools. In practice, we use geodesic modules to bundle terraform code, kops manifests, ansible code, and more.
+
+Geodesic modules are used to logically organize infrastructure as code by leveraging git repositories. We're able to keep code extremely DRY, while at the same time incredibly maintaining modularity and stability. We accomplish this by leveraging a few common patterns:
+
+- [Docker inheritance]({{< relref "tools/docker/best-practices.md#inheritance" >}})
+- [Multi-stage Docker Builds]({{< relref "tools/docker/best-practices.md#multi-stage-builds" >}})
+- [Terraform Root modules]({{< relref "terraform-modules/root/_index.md" >}})
+- [Terraform Modules](/terraform-modules/)
+- [Polyrepos]({{< relref "glossary/polyrepo.md" >}}) to logically organize infrastructure. Use modules to import logic.
+
+We prefer the polyrepo approach to the monorepo approach when it comes in infrastructure. Git repositories allow us to surgically tag/version parts of infrastructure.
+
+For example, we often prescribe organizing repositories by AWS organization. For the purpose of documentation, we'll use `cloudposse.co` as the example organization.
+
+# Reference Architectures
+
+## root.cloudposse.co
+
+The [`root.cloudposse.co`](https://github.com/cloudposse/root.cloudposse.co) module represents an organization's "root" or "top level" AWS account. This is typically your billing organization and handles IAM user account provisioning.
+
+It should also use the `terraform` resource  [`aws_organizations_organization`](https://www.terraform.io/docs/providers/aws/r/organizations_organization.html) to provision all the subaccounts below.
```

Comment:
and [organizations_account](https://www.terraform.io/docs/providers/aws/r/organizations_account.html)